src/App/Pages/TabsPage.cs

```diff
@@ -94,7 +94,14 @@ protected override async void OnAppearing()
                 if (message.Command == "syncCompleted")
                 {
                     Device.BeginInvokeOnMainThread(async () => await UpdateVaultButtonTitleAsync());
-                    await ForcePasswordResetIfNeededAsync();
+                    try
+                    {
+                        await ForcePasswordResetIfNeededAsync();
+                    }
+                    catch (Exception ex)
+                    {
+                        _logger.Value.Exception(ex);
```

Comment:
@andrebispo5 if an error happens here, should the user be redirected somewhere or is it fine to just ignore the force password reset?

Reply:
Like we are doing inside the `ForcePasswordResetIfNeededAsync` method, when there is an error we will only log it.